source/12-maintenance/04-implementation-schedule.rst

```diff
@@ -5,12 +5,23 @@
 Implementation Schedule
 -----------------------
 
-Changes to the COUNTER Code of Practice may be non-substantive or substantive. A non-substantive change may be a clarification or correction of typographical errors that does not affect how the Code of Practice is implemented. A substantive change is one that would affect the implementation of the COUNTER Code of Practice. Examples of substantive changes are adding a new metric type or report, changing the requirement for including a data element from “may” to “MUST”, or changing processing rules.
+Under Explicit Versioning, changes to the COUNTER Code of Practice fall under three headings.
 
-Non-substantive changes can become effective immediately upon publication of the new version of the Code of Practice.
+Fix releases become effective immediately upon publication of the new version of the Code of Practice. Fix releases include bug fixes, typographic amendments and clarifications. 
 
-Substantive changes become effective for a given content provider within 12 months of publication of the new release or with the next audit, whichever date is later.
+Feature releases include backward-compatible and optional changes that become effective six months from the date of publication of the new version of the Code of Practice.
 
-Substantive changes will be clearly marked in the change log in :ref:`Appendix B <appendix-b>` to ensure they can be easily identified.
+COUNTER will put out Fix and/or Feature releases no more than one every six months. That is, if a Fix release went out in October 2023 no further Fix or Feature release would be issued before April 2024.
+
+Breaking releases require report providers to make changes to their implementations of the CoP and report consumers to make changes to their analysis tools. COUNTER therefore does not intend for Breaking releases to happen frequently. From the date of compliance being required, potential Breaking releases will follow this schedule: 
+
+* Compliance required month 1 (e.g. January 2025).
+* Consultation opens for next Breaking release in month 24 (e.g. February 2027).
+* Breaking release goes live month 30 (e.g. September 2027).
+* Compliance required with Breaking release month 48 or subsequent January (e.g. January 2030). 
+
+Report providers may elect to become compliant with Feature and Breaking releases earlier than the stated compliance date.
+
+All changes will be clearly marked in the change log in :ref:`Appendix B <appendix-b>` to ensure they can be easily identified.
```

Comment:
The change log currently is on GitHub. Will it be moved back into the CoP and replace the current appendix B?